apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -436,12 +436,12 @@ export class SubscribersController {
     );
   }
 
-  private toArray(param: string[] | string): string[] | undefined {
-    let paramArray: string[] | undefined;
+  private toArray(param: string[] | string): string[] {
+    let paramArray: string[] = [];
```

Comment:
I wonder if this is the best behavior, since some of the downstream usage might be checking for existence vs length. Did you had a chance to take a look of all the usages?

Reply:
Understood, just saw that we are throwing an exception if it is undefined, will implement the same